applications/ShapeOptimizationApplication/python_scripts/mesh_controller_with_solver.py

```diff
@@ -54,6 +54,11 @@ def __init__(self, MeshSolverSettings, model):
         }""")
         self.MeshSolverSettings = MeshSolverSettings
         self.MeshSolverSettings.ValidateAndAssignDefaults(default_settings)
+        self.MeshSolverSettings["solver_settings"].ValidateAndAssignDefaults(default_settings["solver_settings"])
```

Comment:
doesn't this line create the `mesh_motion_linear_solver_settings` if they are missing?

Reply:
True. I don't know why I added this anymore.